packages/benchmark/testScenarios/singleWebhook/singleWebhook.script.ts

```diff
@@ -0,0 +1,11 @@
+import http from 'k6/http';
+import { check } from 'k6';
+
+const apiBaseUrl = __ENV.API_BASE_URL;
+
```

Comment:
Wondering how we let people know that this can also export the options object, with a plethora of fields configure the behavior 🤔 

<img width="702" alt="image" src="https://github.com/user-attachments/assets/f135c932-0544-4f9b-affd-7d0324c7ecef">

Reply:
Good point. Should write some documentation about that